app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java

```diff
@@ -17,6 +18,7 @@
 import com.cornellappdev.android.eatery.model.DiningHallModel;
 import com.cornellappdev.android.eatery.model.EateryBaseModel;
 import com.cornellappdev.android.eatery.model.enums.PaymentMethod;
+import com.cornellappdev.android.eatery.network.NetworkUtilities;
```

Comment:
might be wrong, but we don't need this import?

Reply:
deleted

Reply:
deleted